docs/data-sources/index.md

```diff
@@ -0,0 +1,92 @@
+# Data Sources
```

Comment:
is it possible to build this automaticaly?

Reply:
I have added an autogeneration in the next commit.